### ISSUE TITLE
CPU usage upto 50% (on 2 core CPU)

### ISSUE THREAD

**Issue by kosz78** [axmor/typecs#4]
There is something inside TypEcs that is easting 1 core of CPU.

see attached pic.

I solved only uninstalling TypeEcs.

Win7 32bit, Java 7, Eclipse 4.4M4 (Enide Studio 2014)

---
- Bitbucket: https://bitbucket.org/axmor/typecs/issue/4
- Originally reported by: Paul Verest
- Originally created at: 2014-03-23T15:26:43.357

**Comment by kosz78** [comment]
_From Konstantin Zaitcev on 2014-03-24 11:44:43+00:00_

Could you please attach your TypeScript project?

**Comment by kosz78** [comment]
_From Paul Verest on 2014-03-31 14:57:55+00:00_

I do not have. I have installed at home and ran into the issue.

At work (Win64) it was OK.

At home I have tried opening a lot of .ts files included the  typescript compiler by MS. That is the biggest one. The TypeScript solution by Palanir would hang for several seconds, but TypEcs was quick and responsive. However I think something in Eclipse or TypEcs was "spoiled" after that.

I cannot now show a way to reproduce.
